coretex/cli/config.py

```diff
@@ -0,0 +1,193 @@
+from pathlib import Path
+
+import os
+
+from tabulate import tabulate
+
+import click
+
+from ..configuration import loadConfig, saveConfig, isUserConfigured, isNodeConfigured
+from ..networking.network_manager import NetworkManager
```

Comment:
Why is this used? `networking.networkManager` object should be used to send requests.